src/components/Certificate.tsx

```diff
@@ -53,58 +53,71 @@ export const CertificateComponent = ({
     document.body.appendChild(a);
     a.click();
     document.body.removeChild(a);
+    setLoading(false);
   };
 
   const handleShareLinkedIn = async () => {
+    if (loading) return;
+    setLoading(true);
     const certificateUrl = `${window.location.origin}/certificate/verify/${certificateSlug}`;
     const postContent = `I just earned the "${course.title}" certificate on 100xDevs! Check it out: ${certificateUrl}`;
     const shareUrl = `https://www.linkedin.com/shareArticle?mini=true&url=${encodeURIComponent(
       certificateUrl,
     )}&text=${encodeURIComponent(postContent)}`;
     window.open(shareUrl);
+    setLoading(false);
   };
 
   const handleShareTwitter = () => {
+    if (loading) return;
+    setLoading(true);
     const tweetText = `I just earned the "${course.title}" certificate on 100xDevs! Check it out:`;
     const certificateUrl = `${window.location.origin}/certificate/verify/${certificateSlug}`;
     const shareUrl = `https://twitter.com/intent/tweet?text=${encodeURIComponent(tweetText)}&url=${encodeURIComponent(certificateUrl)}`;
     window.open(shareUrl);
+    setLoading(false);
   };
 
-  console.log('Url is : ', certificatePdfUrl);
+  if (certificatePdfUrl) {
```

Comment:
remove this please

Reply:
> remove this please

sure, you meant the if check right? or the whole console.log?? @siinghd